packages/@react-spectrum/s2/src/Picker.tsx

```diff
@@ -111,19 +119,42 @@ const inputButton = style<PickerButtonProps | AriaSelectRenderProps>({
   alignItems: 'center',
   height: 'control',
   transition: 'default',
-  columnGap: 'text-to-control',
-  paddingX: 'edge-to-text',
+  columnGap: {
+    default: 'text-to-control',
+    isQuiet: 'text-to-visual'
+  },
+  paddingX: {
+    default: 'edge-to-text',
+    isQuiet: centerPadding()
```

Comment:
isn't this for vertical centering/spacing?

Reply:
hm yeah that was copied over from when we removed isQuiet so not sure why we chose `centerPadding`. i'll just remove it and set the padding to 0 since we don't need any

Reply:
apologies, that might've been me
thank you!